rllib/policy/policy_map.py

```diff
@@ -66,130 +52,102 @@ def __init__(
 
         super().__init__()
 
-        self.worker_index = worker_index
-        self.num_workers = num_workers
-        self.session_creator = session_creator
-        self.seed = seed
+        if any(
+            i is not None
+            for i in [policy_config, worker_index, num_workers, session_creator, seed]
+        ):
+            deprecation_warning(
+                old="PolicyMap([deprecated args]...)",
+                new="PolicyMap(capacity=..., policies_swappable=...)",
+                error=False,
+            )
 
-        # The file extension for stashed policies (that are no longer available
-        # in-memory but can be reinstated any time from storage).
-        self.extension = f".{self.worker_index}.policy.pkl"
+        self.policies_swappable = policies_swappable
 
         # Dictionary of keys that may be looked up (cached or not).
```

Comment:
Nope, good point! Will fix ...

Reply:
Done. Only left `self._lock` as-is, b/c the `_lock` property is used by our `@with_lock` decorator and I didn't want to touch that part of the code in this PR.

Reply:
Actually, changed these all to regular private vars. I would otherwise have no access on these in my test cases (at least on my Mac, it's not working).

Reply:
yep makes sense. I noticed later you need access for unittests.